source/screens/App/shared/const.js

```diff
@@ -1,4 +1,8 @@
 export const MenuItems = [
+  {
```

Comment:
You may not want/need this. There's already the app's logo at the top left. Typically a user would know to click on the branding which will take you to the home page. It's ok if you want it though 👍